index.html

```diff
@@ -8,10 +8,21 @@
     <link rel="stylesheet" href="index.css" />
   </head>
   <body>
+    <header class="head">
+      <div><h1>Awesome Books</h1></div>
+      <nav>
+        <ul class="navbar">
+          <li id="List">List</li>
+          <li id="addNew">Add new</li>
+          <li id="contact">Contact</li>
+        </ul>
+      </nav>
+    </header>
+    <p id="dateTime"></p>
     <div class="collection" id="book-collection">
       <h2 class="book-heading">Awesome Books</h2>
       <div id="book-list"></div>
-      <div class="sec">
+      <div class="sec" id="box">
```

Comment:
When I reload the app, on the list page, the form to add the books is also visible, which should not be the case. I guess you forgot to set the class `hidden` for the form section by default if you want the list to be the main section to be visible when the app is reloaded. Below is the screenshot of your app when I reload it. Kindly fix this issue.
![image](https://user-images.githubusercontent.com/33268927/219010354-ac46ab36-473b-4545-95b9-0a98bddc8a12.png)